src/ape_ethereum/multicall/handlers.py

```diff
@@ -165,6 +177,7 @@ def add(self, call: ContractMethodHandler, *args, **kwargs):
 
         super().add(call, *args, **kwargs)
         self.abis.append(_select_method_abi(call.abis, args))
+        return self
 
     @property
     def returnData(self) -> List[HexBytes]:
```

Comment:
question: how come this is camelCase?

Reply:
To match the name that it has in the contract

I felt like it would be expected when working with that property that it matched, would feel awkward to have to remember it changes slightly

Reply:
Both might be nice, but ya it makes sense..
But also - a comment explaining the veer from the norm would be appreciated

Reply:
For the purposes of the expected usage, it's kinda of "internal", so I don't think we need both

But yes definitely explaining in a comment is a good suggestion

Reply:
```suggestion
    def returnData(self) -> List[HexBytes]:
        # NOTE: this property is kept camelCase to align with the raw EVM struct
```